<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDbCountrySelectorComponent</class>
 <widget class="QFrame" name="CDbCountrySelectorComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>135</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Country selector</string>
  </property>
  <property name="whatsThis">
   <string>select a country</string>
  </property>
  <layout class="QHBoxLayout" name="hl_Country">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="le_CountryIso">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>ISO, &quot;AU&quot;, &quot;US&quot;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_CountryName">
     <property name="placeholderText">
      <string>Name, &quot;Russia&quot;, &quot;France&quot;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_CountryIcon">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
